coordinator/Model/TestSuite.cs

```diff
@@ -155,7 +155,7 @@ public enum TestResultKind {
         TimeLimitExceeded = 4,
         MemoryLimitExceeded = 5,
         ShouldFail = 6,
-        NotRunned = -1,
```

Comment:
External code might depend on these name as they're serialized as JSON (although unlikely). This should be a breaking change